ci/azure-steps.yml

```diff
@@ -43,17 +43,18 @@ steps:
        Set-Content "env:$p" "$v"
      }
 
-     if ($env:compiler -eq 'msvc2015') {
-       if ($env:arch -eq 'x86') {
-         $forcex86 = "--forcex86"
-       }
+     # Install Python3
+     if ($env:arch -eq 'x86') {
+       $forcex86 = "--forcex86"
+     }
 
-       # download and install python3 and add to path (since it's not installed in vs2015 image!)
-       Set-ExecutionPolicy Bypass -Scope Process -Force
-       iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-       choco install python3 -y --no-progress $forcex86 --params "/InstallDir:C:\Python3"
-       $env:Path = "C:\Python3;$env:Path"
+     # download and install python3 and add to path (since it's not installed in vs2015 image!)
```

Comment:
> (since it's not installed in vs2015 image!)
This line is probably not needed anymore

Reply:
Thanks